### PR TITLE
Disable sites in sites.list

### PR DESCRIPTION
**Pull request by joedev1981**
I added a settings page to disable/enable individual sites when the sites.list is loaded. The corresponding code that performs this feature was added to parseView function.

I think this is a great feature to add. If you don't like my implementation but are interested in the feature please feel free to adapt it.

### CODE REVIEW

File: plugin.video.videodevil/resources/settings.xml [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+    <setting id="hide_warning" type="bool" label="30002" default="false"/>
+    <setting id="enable_debug" type="bool" label="20191" default="false"/>
+    <setting type="sep"/>
+    <setting id="video_type" type="enum" lvalues="30013|30014|30015|30016" label="30012" default="0"/>
+    <setting type="sep" />
+    <setting id="download" type="bool" label="30007" default="false"/>
+    <setting id="download_path" type="folder" enable="eq(-1,true)" source="video" label="30008"  option="writeable" default=""/>
+    <setting id="download_ask" type="bool" enable="eq(-2,false)" label="30009" default="false"/>
+  </category>
+  <category label="Folder">
```
Comment (mrdougquaid) [inline plugin.video.videodevil/resources/settings.xml:12]: Use a string ID here and replace "Folder" with something like "Enabled sites"

**Comment by mrdougquaid** [comment]
Merging this will dirty the commit history of the main repo. Get rid of the unnecessary commits in this pull request. There are 14 commits touching a variety of files but only 2 files have changed. I

**Comment by joedev1981** [comment]
Thanks for the feedback. I am traveling at the moment. I will make the changes and submit a pull request as soon as I get home.

File: plugin.video.videodevil/resources/settings.xml (hunk 1 above)
Comment (joedev1981) [inline plugin.video.videodevil/resources/settings.xml:12]: Is there a list of standard string IDs or do we define our own? I wouldn't know how to update other languages other than english.

Comment (mrdougquaid) [inline plugin.video.videodevil/resources/settings.xml:12]: We define our own.

Add 
<string id="30017">Enabled sites</string>
To plugin.video.videodevil/resources/language/English/strings.xml
And reference 30017 in this file

**Comment by mrdougquaid** [comment]
This was committed cleanly in #67

**Pull request closed without merging**